bigquery/data-transfer/api/QuickStart/Program.cs

```diff
@@ -0,0 +1,46 @@
+﻿/*
+ * Copyright (c) 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+// [START bigquery_data_transfer_quickstart]
+
+using System;
+using Google.Api.Gax;
+using Google.Cloud.BigQuery.DataTransfer.V1;
+
+namespace QuickStart
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            // Instantiates a client
+            DataTransferServiceClient client = DataTransferServiceClient.Create();
+
+            // Your Google Cloud Platform project ID
+            string projectId = "YOUR-PROJECT-ID";
+
+            ProjectName project = new ProjectName(projectId);
+            PagedEnumerable<ListDataSourcesResponse, DataSource> sources = client.ListDataSources(ParentNameOneof.From(project));
```

Comment:
How do you like
```
var sources = client.ListDataSources(ParentNameOneof.From(project));
```
?

Reply:
I like it. At first I worried they wouldn't be able to find the types easily, but I think this makes it easier to find the method we are calling so that they can look it up.